### PR TITLE
Add repositories table

### LINKED ISSUE #2: Create repository table

### PR DESCRIPTION
**Pull request by jfontan**
Fixes #2

### CODE REVIEW

File: database.go [hunk 1]
```diff
@@ -22,6 +23,7 @@ type Database struct {
 	ter  sql.Table
 	br   sql.Table
 	or   sql.Table
+	rer  sql.Table
```
Comment (erizocosmico) [inline database.go:26]: can we get some readable table names, please? 🙏

File: repositories.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	rowRepoIter, err := NewRowRepoIter(r.pool, iter)
+	if err != nil {
+		return nil, err
+	}
+
+	return rowRepoIter, nil
+}
+
+func (repositoriesTable) Children() []sql.Node {
+	return []sql.Node{}
```
Comment (erizocosmico) [inline repositories.go]: can we return `nil` instead to save an alloc here?

**Review by erizocosmico — CHANGES_REQUESTED** [review]

File: database.go (hunk 1 above)
Comment (jfontan) [reply database.go:26]: I've created an issue to change all those names:

https://github.com/src-d/gitquery/issues/20

File: repositories.go (hunk 2 above)
Comment (jfontan) [reply repositories.go]: Done, we should also change this in all other tables.

**Review by erizocosmico — APPROVED** [review]

**Review by ajnavarro — APPROVED** [review]

**Pull request merged**